### ISSUE TITLE
Webp support

### ISSUE THREAD

**Issue by MaxSteven** [GeekMagicClock/HelloCubic-Lite#26]
Hi I just got one today.
It's quite cool but can you update so it can support webp gif?
The memory is so low so traditional gif is really limited.
webp can provide smaller file size with better quality :(

**Comment by GeekMagicClock** [comment]
Yes, I have the same idea before, but don't know how to decode and display webp format on the device yet.

**Comment by MaxSteven** [comment]
is it possible to have an opened the bin so other can edit it?

**Comment by GeekMagicClock** [comment]
https://community.platformio.org/t/undefined-reference-errors-when-using-libwebp-library/21026
Sorry no opened bin, but seems possible, I'll try it in my spare time.

**Comment by MaxSteven** [comment]
that's great I just want to help that's all
do you consider to create a product which has bigger size?

**Comment by GeekMagicClock** [comment]
> that's great I just want to help that's all do you consider to create a product which has bigger size?

Yes, Cube-Max, 50x50x50mm size.
https://github.com/GeekMagicClock/Cube-Max

**Comment by MaxSteven** [comment]
I found it. But since it's only 3Mb I hope the next version can allow use micro sd card for larger storage
